teacher.rb

```diff
@@ -0,0 +1,11 @@
+class Teacher < Person
+  attattr_accessor :@specialization
+  def initialize(classroom, name, age, parent_permission, specialization)
+    @specialization = specialization
+    super(classroom, name, age, parent_permission)
+  end
```

Comment:
- [ ] As requested by the previous reviewer, name, and parent_permission should have default values and we should request `specialization` not `classroom`:

  ```
  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
    @specialization = specialization
    super(age, name, parent_permission: parent_permission)
  end
  ```